test/client/audio_consumer_test.py

```diff
@@ -1,171 +1,136 @@
-from mycroft.client.speech.listener import (
-    WakewordExtractor,
-    AudioConsumer,
-    RecognizerLoop
-)
+
+from mycroft.client.speech.listener import AudioConsumer, RecognizerLoop
 from mycroft.client.speech.recognizer_wrapper import (
     RemoteRecognizerWrapperFactory
 )
 
-
 __author__ = 'seanfitz'
 
 
 class MockRecognizer(object):
-    def __init__(self, transcription=None):
+    def __init__(self):
         self.transcriptions = []
 
     def recognize_google(self, audio, key=None, language=None, show_all=False):
-        return self.tanscriptions.pop(0)
+        return self.transcriptions.pop(0)
 
     def set_transcriptions(self, transcriptions):
-        self.tanscriptions = transcriptions
+        self.transcriptions = transcriptions
 
 
 class AudioConsumerTest(unittest.TestCase):
     """
     AudioConsumerTest
     """
+
     def setUp(self):
         self.loop = RecognizerLoop()
         self.queue = Queue()
         self.recognizer = MockRecognizer()
 
         self.consumer = AudioConsumer(
-            self.loop.state,
-            self.queue,
-            self.loop,
-            self.loop.wakeup_recognizer,
-            self.loop.ww_recognizer,
-            RemoteRecognizerWrapperFactory.wrap_recognizer(
-                self.recognizer, 'google'),
-            self.loop.wakeup_prefixes,
-            self.loop.wakeup_words)
+                self.loop.state,
+                self.queue,
+                self.loop,
+                self.loop.wakeup_recognizer,
+                self.loop.mycroft_recognizer,
+                RemoteRecognizerWrapperFactory.wrap_recognizer(
+                        self.recognizer, 'google'))
 
     def __create_sample_from_test_file(self, sample_name):
         root_dir = dirname(dirname(dirname(__file__)))
         filename = join(
-            root_dir, 'test', 'client', 'data', sample_name + '.wav')
+                root_dir, 'test', 'client', 'data', sample_name + '.wav')
         wavfile = WavFile(filename)
         with wavfile as source:
             return AudioData(
-                source.stream.read(), wavfile.SAMPLE_RATE,
-                wavfile.SAMPLE_WIDTH)
+                    source.stream.read(), wavfile.SAMPLE_RATE,
+                    wavfile.SAMPLE_WIDTH)
 
     def test_audio_pos_front_back(self):
-        audio = self.__create_sample_from_test_file('mycroft_in_utterance')
+        audio = self.__create_sample_from_test_file('weather_mycroft')
         self.queue.put(audio)
-        TRUE_POS_BEGIN = 69857 + int(
-            WakewordExtractor.TRIM_SECONDS * audio.sample_rate *
-            audio.sample_width)
-        TRUE_POS_END = 89138 - int(
-            WakewordExtractor.TRIM_SECONDS * audio.sample_rate *
-            audio.sample_width)
-
-        TOLERANCE_RANGE_FRAMES = (
-            WakewordExtractor.MAX_ERROR_SECONDS * audio.sample_rate *
-            audio.sample_width)
+        tolerance = 4000
+        begin = 70000
+        end = 92000
```

Comment:
I think `begin` and `end` should be renamed. There's not really a clear difference between the name of other variable, `pos_begin`, and `begin`.